### ISSUE TITLE
Certificate location error when installing the project on some other computer

### ISSUE THREAD

**Issue by phlisg** [ryoluo/sail-ssl#11]
Hello,

I've installed a project on a new computer and upon launching docker, I noticed that nginx imediately quits.

I see this message in the logs:

```log
2023-06-09 10:45:55 nginx: [emerg] cannot load certificate "/etc/nginx/certs/server.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/nginx/certs/server.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
```

I tried rebuilding from docker-compose, deleted the image and the container, but still nothing. I also tried `sail artisan sail-ssl:install` but reports everything is set correctly.

My docker-compose.yml seems ok too:
```yml
    nginx:
        image: 'nginx:latest'
        ports:
            - '${HTTP_PORT:-8000}:80'
            - '${SSL_PORT:-443}:443'
        environment:
            - SSL_PORT=${SSL_PORT:-443}
            - APP_SERVICE=${APP_SERVICE:-laravel.test}
            - SERVER_NAME=${SERVER_NAME:-localhost}
        volumes:
            - 'sail-nginx:/etc/nginx/certs'
            - './docker/nginx/templates:/etc/nginx/templates'
            - './vendor/ryoluo/sail-ssl/nginx/generate-ssl-cert.sh:/docker-entrypoint.d/99-generate-ssl-cert.sh'
```

What should I do?
Thank you for your help.

**Comment by ryoluo** [comment]
The script to generate sever certificates might be failed to execute.
You should change owner of `vendor` directory to enable to execute the script.
`sudo chown -R ./vendor $user:$user`

**Comment by ryoluo** [comment]
@phlisg 
As resolved in #13, the problem is permision of the script.
Please try `chmod +x vendor/ryoluo/sail-ssl/nginx/generate-ssl-cert.sh`

**Comment by phlisg** [comment]
Thank you, it solved the issue (didn't think it was a permission error but thank you for the heads up :))!
